test/image_test.py

```diff
@@ -561,37 +560,35 @@ def RotateARGBtoRGBA(str_buf):
             test_surface, test_surface, "failing with identical surfaces"
         )
 
-        rgba_buf = pygame.image.tostring(test_surface, "RGBA")
-        rgba_buf = RotateARGBtoRGBA(RotateRGBAtoARGB(rgba_buf))
-        test_rotate_functions = pygame.image.fromstring(
-            rgba_buf, test_surface.get_size(), "RGBA"
-        )
-
-        self._assertSurfaceEqual(
-            test_surface, test_rotate_functions, "rotate functions are not symmetric"
-        )
-
-        rgba_buf = pygame.image.tostring(test_surface, "RGBA")
-        argb_buf = RotateRGBAtoARGB(rgba_buf)
-        test_from_argb_string = pygame.image.fromstring(
-            argb_buf, test_surface.get_size(), "ARGB"
-        )
+        for pair in fmt_combinations:
+            rgba_buf = pygame.image.tostring(test_surface, pair[0])
+            rgba_buf = convert(pair[1], pair[0], convert(pair[0], pair[1], rgba_buf))
+            test_rotate_functions = pygame.image.fromstring(
+                rgba_buf, test_surface.get_size(), pair[0]
+            )
 
-        self._assertSurfaceEqual(
-            test_surface, test_from_argb_string, '"RGBA" rotated to "ARGB" failed'
-        )
+            self._assertSurfaceEqual(
+                test_surface,
+                test_rotate_functions,
+                "converting {} to {} and back is not symmetric".format(
+                    pair[0], pair[1]
```

Comment:
We support python 3.6 and above, so f strings are preferred to use.

Reply:
Yes.
I started with f strings and then noticed a `format()` in existing nearby code so swapped them out :)

![2022-07-31 14_10_13-D__Aa2_temp6_pygame_test_image_test py - SciTE](https://user-images.githubusercontent.com/12681122/182009659-dfbaa6f4-7e24-473b-b3a2-9221315d8eb8.png)